standard-covoiturage_openapi.yaml

```diff
@@ -176,6 +176,43 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
+  /journeys:
+    get:
+      tags:
+      - List
+      summary: Get journeys by their ids.
+      description: Route used to retrieve journeys using their identifiers.
+      operationId: getJourneys
+      parameters:
+        - in: query
+          name: ids[]
```

Comment:
I don't think we should put the brackets in the name, as we have not at other places.